trunk/SUAVE/Input_Output/OpenVSP/vsp_write.py

```diff
@@ -279,17 +278,14 @@ def write_vsp_wing(wing, area_tags, fuel_tank_set_ind, OML_set_ind):
 
     # Twists
     if n_segments != 0:
-        if wing.Segments[0].percent_span_location == 0.:
+        if np.isclose(wing.Segments[0].percent_span_location,0.):
             vsp.SetParmVal( wing_id,'Twist',x_secs[0],wing.Segments[0].twist / Units.deg) # root
```

Comment:
What happens for blended wing bodies when the fuselage is essentially a flying wing and the wing may not be defined close to y = 0

Reply:
This has no effect on the y location. What you're thinking will still work. There is often some rounding with python that ends up with .000000000003 or some garbage like that, this allows that case to be treated like exactly 0.